Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM summerwind/actions-runner:v2.274.1
```

Comment:
I don't know if it's possible but can the base image tag change depending on an `ENV` passed in?

Maybe not and maybe it's too clever, I'm just wondering as `RUNNER_VERSION` is set in the GH workflow as build argument but not used here.

(not a big deal anyway or even a small deal 😬 )